Gems/Atom/Feature/Common/Assets/Materials/Pipelines/LowEndPipeline/ForwardPass.shader.template

```diff
@@ -0,0 +1,43 @@
+{
+    "Source" : "INSERT_AZSL_HERE",
+
+    "DepthStencilState" :
+    {
+        "Depth" :
+        {
+            "Enable" : true,
+            "CompareFunc" : "GreaterEqual"
+        },
+        "Stencil" :
```

Comment:
We will probably need some way to control this eventually, fine as-is

Reply:
Yeah, stencil is one of those things I've had in the back of my mind that needs to be decoupled. I think material pipeline will give us some ability to configure this on a material-type basis at least, through the material pipeline properties and lua script.